de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/session/NegotiationHandler.java

```diff
@@ -0,0 +1,80 @@
+package de.fu_berlin.inf.dpp.server.session;
+
+import org.apache.log4j.Logger;
+import de.fu_berlin.inf.dpp.negotiation.SessionNegotiation;
+import de.fu_berlin.inf.dpp.negotiation.OutgoingSessionNegotiation;
+import de.fu_berlin.inf.dpp.negotiation.IncomingSessionNegotiation;
+import de.fu_berlin.inf.dpp.negotiation.ProjectNegotiation;
+import de.fu_berlin.inf.dpp.negotiation.AbstractOutgoingProjectNegotiation;
+import de.fu_berlin.inf.dpp.negotiation.AbstractIncomingProjectNegotiation;
+import de.fu_berlin.inf.dpp.negotiation.NegotiationTools;
+import de.fu_berlin.inf.dpp.server.progress.ConsoleProgressIndicator;
+import de.fu_berlin.inf.dpp.session.INegotiationHandler;
+import de.fu_berlin.inf.dpp.session.ISarosSessionManager;
+
+public class NegotiationHandler implements INegotiationHandler {
+
+    private static final Logger LOG = Logger
```

Comment:
I have just replicated the style that is used through the whole project for almost every logger. I am not sure, I should change that here, because of consistency reasons.

Reply:
This a refactor TODO of the project. We have to rename the loggers from upper-case to lower-case. Therefore the common style in the project can be misleading :sweat_smile: It would be nice if new loggers already use the lower-case naming.

Reply:
Ah good to know. I will change it.